### DISCUSSION TITLE
_brand.yml logo does not show in html in neither R nor quarto

### DISCUSSION THREAD

**Discussion by carbonmetrics** [quarto-dev/quarto-cli#11599] [Q&A]
I have copied the _brand.yml from the [Quarto website](https://quarto.org/docs/blog/posts/2024-11-25-1.6-release/).

Formatting / typography works for both html and revealjs. 
Logo works only in revealjs, not in html.

**Comment by cscheid** [comment]
Please [read the docs](https://quarto.org/docs/authoring/brand.html). Thanks!
